src/triage/cli.py

```diff
@@ -59,6 +60,81 @@ def stampdb(self, args):
               f"we think your results schema is version {revision} and are upgrading to it")
         stamp_db(revision, args.dbfile)
 
+
+@Triage.register
+class Experiment(Command):
+    """Validate and run experiments, manage experiment database"""
+
+    def __init__(self, parser):
+        parser.add_argument(
+            '-c', '--config',
+            type=argparse.FileType('r'),
+            help="config file for Experiment",
+            required=True
+        )
+        parser.add_argument(
+            '--n-db-processes',
+            type=int,
+            default=1,
+            help="number of concurrent database connections to use"
+        )
+        parser.add_argument(
+            '--n-processes',
+            type=int,
+            default=1,
+            help="number of cores to use"
+        )
+        parser.add_argument(
+            '--project-path',
+            type=str,
+            help="path to store matrices and trained models"
+        )
+        parser.add_argument(
+            '--replace',
+            dest='replace',
+            action='store_true'
+        )
+        parser.set_defaults(
+            validate=True,
+            validate_only=False,
+        )
+
+    @cachedproperty
+    def experiment(self):
+        db_url = self.root.db_url
+        config = yaml.load(self.args.config)
+        db_engine = create_engine(db_url)
+        common_kwargs = {
+            'db_engine': db_engine,
+            'project_path': self.args.project_path,
+            'config': config,
+            'replace': self.args.replace,
+        }
+        if self.args.n_db_processes > 1 or self.args.n_processes > 1:
+            experiment = MultiCoreExperiment(
+                n_db_processes=self.args.n_db_processes,
+                n_processes=self.args.n_processes,
+                **common_kwargs
+            )
+        else:
+            experiment = SingleThreadedExperiment(**common_kwargs)
+        return experiment
+
+    @cmdmethod('-v', '--validate', action='store_true', help="validate before running audition")
+    @cmdmethod('--no-validate', action='store_false', dest='validate', help="run experiment without validation")
+    @cmdmethod('--validate-only', action='store_true', help="only validate the config file not running Experiment")
```

Comment:
FYI, it looked like in your example you wanted to be able to say:

    triage experiment --no-validate

But you've only defined specification of that option on `triage experiment run`.

I'd say either update the examples or perhaps just simplify this and get rid of the `run` subcommand, (at least for now). You have another subcommand, `configversion`, but I wonder if it really belongs under `experiment` either; it can safely go under the `triage` base command.